arch/arm/arm_features.c

```diff
@@ -16,7 +16,7 @@
 #  endif
 #  include <sys/sysctl.h>
 #elif defined(_WIN32)
-#  include <winapifamily.h>
+#  include <Windows.h>
```

Comment:
Can you change this to lowercase w like we use elsewhere?

Reply:
While that may be the case, the lowercase argument is better especially when it comes to cross compiling from a *nix host where case sensitivity would play a role, I think that's the main reasoning for the ask.

Reply:
I am only asking for consistency.

Reply:
Sure if your toolchain headers follow convention, but when it comes to mingw windows.h is lower case, and any project making use of say Windows.h causes an issue on case sensitive file systems such as ext4 etc. Windows itself being case insensitive, it doesn't care about such matters.